au.yaml

```diff
@@ -151,6 +151,12 @@ months:
     wday: 1
     year_ranges:
       - before: 2017
+  - name: Day of mourning for Queen Elizabeth II
+    regions: [au, au_nsw, au_act, au_sa, au_tas, au_vic, au_wa, au_nt, au_qld]
```

Comment:
Can't you just add this to `au` ?

Reply:
Not that it should matter

Reply:
Was just trying to catch if customers had just state set up.